lib/modules/platform/github/index.ts

```diff
@@ -1731,6 +1732,28 @@ export async function updatePr({
   }
 }
 
+export async function reattemptPlatformAutomerge({
+  prNo,
+  platformOptions,
+}: ReattemptPlatformAutomergeConfig): Promise<void> {
+  try {
+    const { body: ghPr } = await githubApi.getJson<GhRestPr>(
+      `repos/${config.parentRepo ?? config.repository}/pulls/${prNo}`,
+    );
+    const result = coerceRestPr(ghPr);
```

Comment:
```suggestion
    const result = getPr(prNo);
```
this will fetch from cache when available and uses rest if not on cache. so you probably don't need to change tests.